app/assets/javascripts/utilities/buildArticleHTML.js.erb

```diff
@@ -49,7 +49,7 @@ function buildArticleHTML(article) {
     var profileUsername = article.user.username
     var orgHeadline = "";
     if (article.organization && !document.getElementById("organization-article-index")) {
-      orgHeadline = '<div class="article-organization-headline"><a class="org-headline-filler" href="/'+article.organization.slug+'"><span class="article-organization-headline-inner"><img alt="'+article.organization.name+' logo" src="'+article.organization.profile_image_90+'" loading="lazy">'+article.organization.name+'</span></a></div>'
+      orgHeadline = '<div class="article-organization-headline"><a class="org-headline-filler" href="/'+article.organization.slug+'"><img alt="'+article.organization.name+' logo" src="'+article.organization.profile_image_90+'" loading="lazy">'+article.organization.name+'</a></div>'
```

Comment:
This is implemented here and also in the server at `app/views/articles/_single_story.html.erb` and a similar card is rendered in notifications... `app/views/notifications/_article.html.erb`

Reply:
I've just checked - I have actually changed it in these files already. Is there any specific reason why you brought it up?

Reply:
These changes will also need to be ported to the new Preact components that haven't been merged yet.